orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -480,10 +488,18 @@ def testInstantiateORTTrainer(step_fn):
     model = pt_model.TransformerModel(28785, 200, 2, 200, 2, 0.2)
     my_loss = ort_utils.my_loss
     model_desc = ort_utils.transformer_model_description()
-    optim_config = optim.LambConfig()
-
-    # Create ORTTrainer
-    trainer = orttrainer.ORTTrainer(model, model_desc, optim_config, loss_fn=my_loss)
+   
+    max_train_step = 1
+    if lr_scheduler:
+        max_train_step = 10
+        optim_config = optim.SGDConfig(lr=1)
+        opts = orttrainer.ORTTrainerOptions({'lr_scheduler' : lr_scheduler(max_train_step, 0.5)})
+        # Create ORTTrainer
+        trainer = orttrainer.ORTTrainer(model, model_desc, optim_config, loss_fn=my_loss, options=opts)
```

Comment:
we could create an empty dict like `options={}` outside the `if lr_scheduler` and update it to `{'lr_scheduler' : lr_scheduler(max_train_step, 0.5)}`.

with this, you can move `trainer = orttrainer.ORTTrainer(model, model_desc, optim_config, loss_fn=my_loss, options=opts)` outside the if